### ISSUE TITLE
Check that form elements behave as expected

### ISSUE THREAD

**Issue by Angelsaur** [alan-turing-institute/AutSPACEs#57]
### Summary

When you were working your way through some of your sample forms, you should have noticed whether interacting with any of the elements caused anything unexpected to happen
This includes things like a dropdown menu or radio button (an element that lets you select just one of a few options) causing a new page to open when you click on it.

### Deadline 

20/02/2021

### What needs to be done?

Check forms to ensure this isn't an issue
